osu.Game/Screens/Edit/Compose/Components/EditorSelectionHandler.cs

```diff
@@ -104,12 +164,48 @@ protected virtual void UpdateTernaryStates()
             {
                 bindable.Value = GetStateFromSelection(SelectedItems, h => h.Samples.Any(s => s.Name == sampleName));
             }
+
+            foreach ((string bankName, var bindable) in SelectionBankStates)
+            {
+                bindable.Value = GetStateFromSelection(SelectedItems, h => h.SampleControlPoint.SampleBank == bankName);
+            }
         }
 
         #endregion
 
         #region Ternary state changes
 
+        /// <summary>
+        /// Adds a sample bank to all selected <see cref="HitObject"/>s.
+        /// </summary>
+        /// <param name="bankName">The name of the sample bank.</param>
+        public void AddSampleBank(string bankName)
+        {
+            EditorBeatmap.PerformOnSelection(h =>
+            {
+                if (h.SampleControlPoint.SampleBank == bankName)
+                    return;
+
+                h.SampleControlPoint.SampleBank = bankName;
+                EditorBeatmap.Update(h);
+            });
+        }
+
+        /// <summary>
+        /// Removes a sample bank from all selected <see cref="HitObject"/>s.
+        /// </summary>
+        /// <param name="bankName">The name of the sample bank.</param>
+        public void RemoveSampleBank(string bankName)
```

Comment:
Method is not actually used. Not sure it makes any sense either?

Reply:
I've removed this.